libs/frontend/ui-build/src/components/BuildDndOverlay.vue

```diff
@@ -95,7 +95,8 @@ const hoverOverlayStyle = computed(() => {
       return {
         top: `${top}px`,
         left: `${left}px`,
-        width: `${width}px`,
+        // -1 to prevent horizontal scrollbar from showing up.
+        width: `${width - 1}px`,
```

Comment:
I noticed there'll be a horizontal scrollbar at the bottom when the hover overlay is visible:

https://github.com/samatechtw/pubstudio-builder/assets/59676941/3586fa30-5087-4d4e-b174-fe8af946e90c

I couldn't remember seeing this before 🤔. The `-1` here seems to fix it, and is not too noticeable.

Reply:
Good catch!